src/www/ui/api/documentation/openapi.yaml

```diff
@@ -869,77 +869,6 @@ paths:
                 $ref: '#/components/schemas/User'
         default:
             $ref: '#/components/responses/defaultResponse'
-    put:
-      operationId: modifyUserById
```

Comment:
I am hoping this removal is due to merge conflicts. Please make sure the endpoint is not removed :-)

Reply:
sure sir 👍

Reply:
@dushimsam , the `put` operation is still missing in the PR.

Please have a look!

Reply:
> @dushimsam , the `put` operation is still missing in the PR.
> 
> Please have a look!

got it sorry i messed up